config.js

```diff
@@ -28,12 +28,12 @@ var config = module.exports.config = {};
  * @param  {String}    path            The path to the UI static assets
  */
 config.ui = {
-    'path': '../3akai-ux'
+    'path': './3akai-ux'
```

Comment:
These changes might also cause problems with the puppet config... Paging @davidoae to give them a look over.